pkg/testrunner/template/values.go

```diff
@@ -156,6 +156,9 @@ func (r *shootValueRenderer) GetValues(shoot *common.ExtendedShoot, defaultValue
 	if shoot.AdditionalAnnotations != nil {
 		values["shoot"].(map[string]interface{})["shootAnnotations"] = util.MarshalMap(shoot.AdditionalAnnotations)
 	}
+	if shoot.AdditionalLocations != nil && len(shoot.AdditionalLocations) > 0 {
```

Comment:
should we rather add the additional location like we do with the component descriptor?
So directly add them directly on the internal testrun instead of relying on helm. 
see https://github.com/gardener/test-infra/blob/master/pkg/testrunner/template/renderer.go#L93

With the current implementation we would also require every testrun to implement the `range` op in its charts.

WDYT?